dispatcher/backend/src/common/languages.py

```diff
@@ -0,0 +1,337 @@
+""" static list of languages supported by zimfarm
```

Comment:
Ummm... Is it really necessary to list all languages here? Also, once we add this here, we'd have to maintain it to be up to date. 
I think the one and only central place to store languages we support should be the database. We can do a `$group` to get it.

Reply:
Agrees. As I mentioned, I started this way and came back to this to ease UI integration.
Will update the PR.